grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -46,20 +47,12 @@ class JSONRequest(werkzeug_wrappers_json.JSONMixin, werkzeug_wrappers.Request):
 class JSONResponse(werkzeug_wrappers_json.JSONMixin,
                    werkzeug_wrappers.Response):
 
-  def __init__(
-      self,
-      response=None,
-      status=None,
-      headers=None,
-      mimetype=None,
-      content_type=None,
-      direct_passthrough=False,
-  ):
-    try:
-      json_response = json.dumps(response)
-    except TypeError:
-      json_response = response
-    super().__init__(response=json_response, mimetype=JSON_MIME_TYPE)
+  def __init__(self, *args, **kwargs) -> None:
```

Comment:
Specify `response=None` as first argument before *args and **kwargs to support the common `Response` initialization where the contents are passed as first argument.